client/components/Pagination.jsx

```diff
@@ -0,0 +1,93 @@
+import styled, { css } from 'styled-components';
+
+const Pagination = (props) => {
+  const {
+    pagingAndSorting,
+    noOfRatings,
+    setPagination
+  } = props;
+  const { currentPage, offset } = pagingAndSorting;
```

Comment:
This can be merged into the earlier destructuring like below:

pagingAndSorting { currentPage, offset },